### ISSUE TITLE
Switch: d-switch-leading and d-switch-trailing mispositioned in RTL mode

### ISSUE THREAD

**Issue by wkeese** [ibm-js/deliteful#454]
The switch has two divs (`d-switch-leading` and `d-switch-trailing`) to control the color before and after the knob.  That works fine in LTR mode, but the problem is that in RTL mode, `d-switch-leading` appears after (i.e. to the left of) the knob, and `d-switch-trailing` appears before (i.e. to the right of) the knob.   Thus, if the app wants to override the color before or after the knob, it needs to have special CSS rules for RTL mode like:

```css
.d-switch.d-rtl .d-switch-leading,
.d-switch[dir="rtl"] .d-switch-leading {
  background: #eeeeee;
  font-weight: bold;
  color: #555555;
}
```

Instead of this, it would be better for the widget's built-in CSS to properly position the `d-switch-leading` and `d-switch-trailing` divs in RTL mode.

In LTR mode the divs are positioned using `left: 50%` and `right: 50%`:

```css
.d-switch-leading {
	...
	right: 50%;
	...
}

.d-switch-trailing {
	...
	left: 50%;
	...
}
```

In RTL mode, the widget should do the opposite:

```css
.d-switch-leading.d-rtl {
	left: 50%;
        right: auto;
}

.d-switch-trailing.d-rtl {
	right: 50%;
	left: auto;
}
```

Relatedly, fixing this would eliminate the overly complex code in the template that branches based on RTL vs. LTR:

```html
<div class="-d-switch-block d-switch-leading d-switch-width">
{{this.isLeftToRight() ? this.checkedLabel : this.uncheckedLabel}}&nbsp</div>
```

It just becomes:

```html
<div class="-d-switch-block d-switch-leading d-switch-width">
{{this.checkedLabel}}</div>
```

**Comment by wkeese** [comment]
PS: The CSS for Switch is also wrong because it's using `dir=rtl` whereas it should be using the `d-rtl` class.   See ibm-js/delite#362.

**Comment by wkeese** [comment]
Fixed by 0f9cf7e1de6a71d6aa4e0596d44c40651873c422.
